src/core/agent/services/keriaNotificationService.ts

```diff
@@ -356,64 +357,94 @@ class KeriaNotificationService extends AgentService {
       return false;
     }
 
-    if (
-      telStatus === Ilks.rev &&
-      existingCredential &&
-      existingCredential.status !== CredentialStatus.REVOKED
-    ) {
-      await this.credentialService.markAcdc(
-        exchange.exn.e.acdc.d,
-        CredentialStatus.REVOKED
-      );
-      await this.ipexCommunications.createLinkedIpexMessageRecord(
-        exchange,
-        ConnectionHistoryType.CREDENTIAL_REVOKED
-      );
+    if (telStatus === Ilks.rev) {
+      if (!existingCredential) {
```

Comment:
We can simplify and always call this, regardless if the credential exists or not